README.md

```diff
@@ -104,6 +109,8 @@ commands should always be `load` and `save`, respectively.
 
 [appveyor]: https://ci.appveyor.com/project/yitzchak/dicy/branch/master
 
+[arara]: http://ctan.org/pkg/arara
+
 [asymptote]: http://asymptote.sourceforge.net/
 
 [biber]: http://biblatex-biber.sourceforge.net/
```

Comment:
Link to CTAN (https://www.ctan.org/pkg/biber) can be an option like arara or latexmk. CTAN page links to the SourceForge page.